doc/nrf/protocols/matter/overview/group_communication.rst

```diff
@@ -0,0 +1,268 @@
+.. _ug_matter_group_communication:
+
+Matter Group Communication
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section describes how Group Communication works in Matter.
+
+Group Communication in the Matter standard refers to the ability of devices within a Matter network to communicate with multiple devices simultaneously using multicast messages.
+This is particularly useful for scenarios where a command needs to be sent to multiple devices at once, such as turning off all lights in a house or setting a scene.
+
+Group setup
+***********
+
+Devices are configured to be part of one or more groups.
+Each group is identified by a unique group ID.
+A device can be a member of multiple groups, and a group can include multiple devices.
+
+Group Messaging
+***************
+
+When a command needs to be sent to all devices in a group, the command initiator (a controller, a smartphone app, or another device) sends a multicast message addressed to the group ID.
+Only the devices that are members of that group will receive messages to or act on a message.
+Using Group Communication reduces network traffic because a single multicast message can reach multiple devices.
+This is more efficient than sending individual messages to each device.
+It also enhances scalability as adding more devices to a group does not increase the number of messages required for group-wide commands.
+
+.. note::
+
+   Group Communication is by design not reliable, because it does not use any way of acknowledgment.
+   Because of that, the device that sends the message cannot determine whether it reached all devices belonging to the group.
+   Communication reliability depends on the underlying networking technology.
+   For example, sleepy devices should not be targeted in the group messaging, as they will likely miss incoming messages, because their radio is inactive for most of the time.
+   Multicast messages are not buffered by their parents, and are sent when the device is active, so they will not reach the destination.
```

Comment:
This is not right, they are not buffered, so they cannot be sent.
```suggestion
   Multicast messages are not buffered by their parents, so they will not reach the destination.
```

I have impression that I tried to introduce to many information and it mixed things up a bit :) Generally what I've meant is that parent sends buffered messages when the sleepy device is active. But in case of multicasts the messages are not buffered, so it won't work. Maybe it's too detailed to write all of these, so removing just part of the sentence above should work.